src/main/java/tc/oc/pgm/observers/tools/GamemodeTool.java

```diff
@@ -0,0 +1,90 @@
+package tc.oc.pgm.observers.tools;
+
+import com.google.common.collect.Lists;
+import java.util.List;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.HoverEvent;
+import org.apache.commons.lang.WordUtils;
+import org.bukkit.GameMode;
+import org.bukkit.Material;
+import tc.oc.component.Component;
+import tc.oc.component.render.ComponentRenderers;
+import tc.oc.component.types.PersonalizedText;
+import tc.oc.component.types.PersonalizedTranslatable;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.gui.InventoryGUI;
+import tc.oc.pgm.observers.ObserverTool;
+
+public class GamemodeTool implements ObserverTool {
+
+  @Override
+  public Component getName() {
+    return new PersonalizedTranslatable("observer.tools.gamemode");
+  }
+
+  @Override
+  public ChatColor getColor() {
+    return ChatColor.DARK_AQUA;
+  }
+
+  @Override
+  public List<String> getLore(MatchPlayer player) {
+    Component gamemode =
+        new PersonalizedText(WordUtils.capitalizeFully(player.getGameMode().name().toLowerCase()))
```

Comment:
I'm sure there is a translation for this in the client for thìs. Can we use it?

Reply:
Curious. I’m not familiar with the client translations, could you explain?

Reply:
These are the values in the 1.8.9 en_US.lang file:
```
gameMode.survival=Survival Mode
gameMode.creative=Creative Mode
gameMode.adventure=Adventure Mode
gameMode.spectator=Spectator Mode
```
Alternatively:
```
selectWorld.gameMode.survival=Survival
selectWorld.gameMode.creative=Creative
selectWorld.gameMode.adventure=Adventure
selectWorld.gameMode.spectator=Spectator
```

Reply:
Thanks @Meeples10!
Ah I understand now, that’s very useful 👍

Reply:
Have you confirmed that this actually works? I'm not sure if PGM translates vanilla messages.